enterprise/frontend/src/metabase-enterprise/api/ai-streaming/process-stream.ts

```diff
@@ -248,9 +248,7 @@ export async function processChatResponse(
         config.onToolResultPart?.(streamPart.value);
       }
       if (streamPart.name === "error") {
-        config.onError?.(
-          new Error(streamPart.value ? `${streamPart.value}` : "Unknown error"),
-        );
+        config.onError?.(streamPart.value);
```

Comment:
redux serializes the `Error` so we ended up with an issue where the data in the error is not easily accessible / needs to be parsed.